pkg/cmd/root/root.go

```diff
@@ -56,14 +58,17 @@ func NewCmdRoot(f factory.Factory, clientFactory apiclient.ClientFactory, askPro
 	cmd.AddCommand(environmentCmd.NewCmdEnvironment(f))
 
 	// configuration commands
+	cmd.AddCommand(configCmd.NewCmdRelease(f))
 	cmd.AddCommand(spaceCmd.NewCmdSpace(f))
 
 	cmd.AddCommand(releaseCmd.NewCmdRelease(f))
 
+	viper.BindPFlag(constants.ConfigNoPrompt, cmdPFlags.Lookup(constants.FlagNoPrompt))
```

Comment:
question: Does this take the value from the viper config file, and stick it into the pflag? or does it go the other way?

Once we have a viper flag bound to a pflag, should we be using viper to read the value, or the pflag?

The viper help is predictably useless: https://pkg.go.dev/github.com/spf13/viper#BindPFlag

Reply:
They way it works is when the p flag is bound viper will use it if the value is set. Viper reads the pflag when you call `viper.get` for the key we bind too. Viper will check if the key we request has a bounded pflag first, after that if not flag value is set it will check env, lastly is checks the config file.

Reply:
So yes, if we bind a pflag in viper, we should be using viper to request that value